diff -Naur dyld_origin/include/mach-o/dyld.h dyld/include/mach-o/dyld.h
--- dyld_origin/include/mach-o/dyld.h	2021-05-06 11:08:50.000000000 +0800
+++ dyld/include/mach-o/dyld.h	2021-05-06 11:13:17.000000000 +0800
@@ -41,6 +41,9 @@
  #define DYLD_DRIVERKIT_UNAVAILABLE
 #endif

+#define __API_UNAVAILABLE_PLATFORM_bridgeos bridgeos,unavailable
+#define __API_AVAILABLE_PLATFORM_bridgeos(x) bridgeos,introduced=x
+
 /*
  * The following functions allow you to iterate through all loaded images.
  * This is not a thread safe operation.  Another thread can add or remove
